.github/workflows/release.yml

```diff
@@ -55,8 +55,7 @@ jobs:
 
       # Push to Ballerina Central
       - name: Ballerina Push
-        run: bal push
+        run: cd ./ballerina && bal push
```

Comment:
We can add `working-directory: ./ballerina` as an alternative for the environment variable